### PR TITLE
[Merged by Bors] - chore(SetTheory): remove use of autoImplicit

### PR DESCRIPTION
**Pull request by grunweg**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/SetTheory/Game/PGame.lean [hunk 1]
```diff
@@ -301,26 +297,28 @@ macro "pgame_wf_tac" : tactic =>
 -- Register some consequences of pgame_wf_tac as simp-lemmas for convenience
 -- (which are applied by default for WF goals)
 
+variable {xl xr : Type u} {xR : xr → PGame}
```
Comment (eric-wieser) [inline Mathlib/SetTheory/Game/PGame.lean:300]: This seems a bit inconsistent; can you move `xR` to the  two lemmas that use it?

File: Mathlib/SetTheory/Ordinal/NaturalOps.lean [hunk 2]
```diff
@@ -120,12 +117,12 @@ theorem toOrdinal_eq_one (a) : toOrdinal a = 1 ↔ a = 1 :=
...
 
 @[simp]
-theorem toOrdinal_max : toOrdinal (max a b) = max (toOrdinal a) (toOrdinal b) :=
+theorem toOrdinal_max {a b : NatOrdinal} : toOrdinal (max a b) = max (toOrdinal a) (toOrdinal b) :=
   rfl
 #align nat_ordinal.to_ordinal_max NatOrdinal.toOrdinal_max
 
 @[simp]
-theorem toOrdinal_min : toOrdinal (min a b)= min (toOrdinal a) (toOrdinal b) :=
+theorem toOrdinal_min {a b : NatOrdinal} : toOrdinal (min a b)= min (toOrdinal a) (toOrdinal b) :=
```
Comment (eric-wieser) [inline Mathlib/SetTheory/Ordinal/NaturalOps.lean:125]:
```suggestion
theorem toOrdinal_min {a b : NatOrdinal} : toOrdinal (min a b) = min (toOrdinal a) (toOrdinal b) :=
```

**Review by eric-wieser — APPROVED** [review]
bors d+

**Comment by grunweg** [comment]
Thank you for the quick review!
bors r+

**Pull request closed without merging**